tools/distrib/python/grpcio_tools/setup.py

```diff
@@ -95,7 +95,10 @@ def protoc_ext_module():
           protoc_lib_deps.CC_INCLUDE,
       ],
       language='c++',
-      define_macros=[('HAVE_PTHREAD', 1)],
+      define_macros=[
+          ('HAVE_PTHREAD', 1,),
+          ('GOOGLE_PROTOBUF_NO_THREADLOCAL', 1,),
```

Comment:
Your right, I was thinking about the protobuf runtime but I'm fine with it if its for code generation.